PSDepend/PSDepend.psd1

```diff
@@ -4,7 +4,7 @@
 RootModule = 'PSDepend.psm1'
 
 # Version number of this module.
-ModuleVersion = '0.1.63'
+ModuleVersion = '0.2.0'
```

Comment:
Are you going to release a release candidate or beta?

Reply:
Hiyo!

Probably not - thought process:

* This just adds a bit to validate platform support of independent scripts
* We're not saying 'xplat is a thing!' - will just mention that it works in some very limited, specific environments and that we'll update and ping #11 as we work on xplat support and functionality
* I don't have time : )  Makes sense for a large project like Pester, but I barely have time to give the okay for other folks' code

It's definitely a cool idea, but for this particular PR I'm happy enough to just bump the version

Cheers!

Reply:
The just merged changes are now available via PSGallery as version 0.1.66 and not 0.2.0. Is that correct?

Reply:
Correct but unintended : )  Using different build setups per module sometimes bites me, assumed I had the psd1-can-override-version-bump login in place... going to add some release notes and bump for a release now

Reply:
Whew!  worked - https://www.powershellgallery.com/packages/PSDepend/0.2.0

Reply:
Probably completely unnecessary, but unlisted 0.1.66, 0.1.67 that had these changes without the version bump, things should be in order now.

Cheers!

Reply:
Cool!